media_stack.py

```
@@ -0,0 +1,102 @@
+import http.client
+import os
+import urllib.parse
+import smtplib
```

Ruff (0.8.2)

`smtplib` imported but unused

Remove unused import: `smtplib`

(F401)